docs/recipes/any_fixture.md

```diff
@@ -90,6 +100,10 @@ before(:all) { fixture(:account) }
 
 # You can also use it to fetch the record (instead of storing it in instance variable)
 let(:account) { fixture(:account) }
+
+# You can just use `before_reset` or `after_reset` callbacks
+before_reset { Post.delete_all }
```

Comment:
Let's make the names of the methods more explicit, say, `before_fixtures_reset`

Reply:
Renamed them 🙂